File: test/python/framework/tutorials/tutorial_06_wdd.py
import os

# Setup mesh
N = 10
L = 2.0
xmin = -L / 2
dx = L / N
nodes = [xmin + k * dx for k in range(N + 1)]
meshgen = OrthogonalMeshGenerator(node_sets=[nodes, nodes])
grid = meshgen.Execute()
grid.SetUniformBlockID(0)

SimTest06_WDD(grid)
MPIBarrier()
if rank == 0:
    os.system("rm SimTest_06*")

File: test/lua/framework/tutorials/tutorial_06_wdd.lua
-- Setup mesh
if nmesh == nil then
  nmesh = 10
end

nodes = {}
N = nmesh
L = 2.0
xmin = -L / 2
dx = L / N
for i = 1, (N + 1) do
  k = i - 1
  nodes[i] = xmin + k * dx
end

meshgen1 = mesh.OrthogonalMeshGenerator.Create({ node_sets = { nodes, nodes } })
grid = meshgen1:Execute()

-- Set block IDs
grid:SetUniformBlockID(0)

unit_sim_tests.SimTest06_WDD({ mesh = grid })
MPIBarrier()
if location_id == 0 then
  os.execute("rm SimTest_06*")
end
